README.md

```diff
@@ -1,17 +1,119 @@
+[![Total Downloads](https://img.shields.io/packagist/dt/laravel-notification-channels/unifonic.svg?style=flat-square)](https://packagist.org/packages/laravel-notification-channels/unifonic)
 
-## Workflow for new channels
+This package makes it easy to send notifications using [Unifonic Api](https://unifonic.docs.apiary.io/#reference/messages/send) with Laravel 5.5+, 6.x, 7.x and 8.x
 
-1) Head over to the [skeleton repo](https://github.com/laravel-notification-channels/skeleton) download a ZIP copy. This is important, to ensure you start from a fresh commit history.
-2) Use find/replace to replace all of the placeholders with the correct values (package name, author name, email, etc).
-3) Implement to logic for the channel & add tests.
-4) Fork this repo, add it as a remote and push your new channel to a branch.
-5) Submit a new PR against this repo for review.
 
-Take a look at our [FAQ](http://laravel-notification-channels.com/) to see our small list of rules, to provide top-notch notification channels.
+
+## Contents
+- [Requierements](#requirements)
+- [Installation](#installation)
+	- [Setting up the Unifonic service](#setting-up-the-Unifonic-service)
+- [Usage](#usage)
+	- [Available Message methods](#available-message-methods)
+- [Changelog](#changelog)
+- [Testing](#testing)
+- [Security](#security)
+- [Contributing](#contributing)
+- [Credits](#credits)
+- [License](#license)
+
+
+## Requierements
+- Before start you have to create an account on [Unifonic](https://unifonic.com).
+
+## Installation
+
+1. You can install the package via composer:
+
+``` bash
+composer require laravel-notification-channels/unifonic
+```
+This package will register itself automatically if your Laravel 5.5+, trough Package auto-discovery.
+
+
+### Setting up the Unifonic service
+
+Add your Unifonic AppsId to your `config/services.php`:
+
+```php
+// config/services.php
+...
+'unifonic' => [
+    'appsId' => env('UNIFONIC_APPS_ID'),
+],
+...
+```
+
+## Usage
+
+
+Now you can use the channel in your `via()` method inside the notification:
+
+``` php
+use NotificationChannels\Unifonic\UnifonicChannel;
+use NotificationChannels\Unifonic\UnifonicMessage;
+use Illuminate\Notifications\Notification;
+
+class InvoicePaid extends Notification
+{
+    public function via($notifiable)
+    {
+        return [UnifonicChannel::class];
+    }
+
+    public function toUnifonic($notifiable)
+    {
+        return UnifonicMessage::create("Your invoice {$notifiable->order->id} was paid!");
+    }
+}
+```
+
+In order to let your Notification know which phone numer you are targeting, add the `routeNotificationForUnifonic` method to your Notifiable model.
+
+**Important note**: Unifonic requires the recipients phone number to be without `+` like this format. 21267064497
```

Comment:
Could say e164, and just strip the leading plus if it's set
https://developers.omnisend.com/guides/e164-phone-number-formatting